Cloud Application Manager/Cloud Optimization/partner-cloud-integration-aws-ri.md

```diff
@@ -0,0 +1,79 @@
+{{{
+  "title": "Partner Cloud Integration: CenturyLink AWS Reserved Instance Strategy",
+  "date": "2-7-2018",
+  "author": "Andy Watson",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Overview
+
+CenturyLink [Cloud Application Manager's](https://www.ctl.io/cloud-application-manager/) Cloud Optimization offers [consolidated billing](https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration-consolidated-billing/) to make payments easy. Part of the consolidated billing involves CenturyLink's strategy for billing Amazon Web Services [Reserved Instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-reserved-instances.html).
+
+
+### Audience
+
+Users of accounts that already have or are considering CenturyLink Optimization of AWS Accounts. This is for all new standard AWS accounts created through Cloud Application Manager and any existing account moving into CenturyLink's care who have bought Amazon Web Services Reserved Instances.
+
+
+### Prerequisites
+
+For a new AWS Account:
+
+* The customer must have reviewed the process for creating a [new Amazon Web Services account](https://www.ctl.io/knowledge-base/cloud-application-manager/cloud-optimization/partner-cloud-integration-aws-new/) and is thinking about buying AWS Reserved Instances.
```

Comment:
Change link to `(partner-cloud-integration-aws-new.md)`

Reply:
Update completed.